Services/ProductAttributeService.cs

```diff
@@ -33,7 +33,7 @@ public class ProductAttributeService : IProductAttributeService {
 
         public IEnumerable<ProductAttributePart> GetAttributes(IEnumerable<int> ids) {
             var attributes = AttributesById;
-            return ids.Select(id => attributes[id]).ToList();
+            return ids.Where(id => attributes.ContainsKey(id)).Select(id => attributes[id]).ToList();

```

Comment:
Looks like I based this branch off the changes I made for #91. This comes from that.